roles/openshift_provisioners/tasks/generate_clusterrolebindings.yaml

```diff
@@ -1,6 +1,8 @@
 ---
 - name: Generate ClusterRoleBindings
-  template: src=clusterrolebinding.j2 dest={{mktemp.stdout}}/templates/{{obj_name}}-clusterrolebinding.yaml
+  template:
+    src: clusterrolebinding.j2
+    dest: {{mktemp.stdout}}/templates/{{obj_name}}-clusterrolebinding.yaml
```

Comment:
Missing spaces and quotes here.  This line should be

```yaml
dest: "{{ mktemp.stdout }}/templates/{{ obj_name }}-clusterrolebinding.yaml"
```

Other files need similar changes.